packages/block-library/src/social-links/block.json

```
@@ -10,7 +10,7 @@
 		"iconColor": {
 			"type": "string"
 		},
-		"customIconColor": {
+		"customIconColor": {	
```

```suggestion
		"customIconColor": {
```
Don't know why the linter doesn't complain about this but let's get rid of the tab.